pkg/pb/contacts.override.gorm.go

```diff
@@ -0,0 +1,28 @@
+package pb
+
+// BeforeToORM ...
```

Comment:
if this is hand-written code let's not set a bad example with lazy comments like this :)